app/utils/google-maps.js

```diff
@@ -0,0 +1,32 @@
+/*globals google*/
+import Ember from 'ember';
+
+let geocoder = new google.maps.Geocoder();
```

Comment:
I tried to extract the google dependencies into a utils file, so I can more easily stub them for tests